geopy/geocoders/googlev3.py

```diff
@@ -216,8 +222,18 @@ def geocode(
         params = {
             'sensor': str(sensor).lower()
         }
-        if query is None and not components:
+
+        if not any([query, components, place_id]):
             raise ValueError('Either `query` or `components` must be set.`')
+
+        if place_id and (bounds or query):
```

Comment:
Looks like this should include the other 2 options as well.

I guess this would do it:
```
if sum(1 for value in (query, components, place_id) if value) != 1:
```

Reply:
Oh, I didn't notice that it's `bounds` instead of `components`. Is that correct? If so, should the both conditions be `or`'ed: the yours and mine?

Reply:
When the ``place_id`` parameter is used, it cannot be used with the ``bounds`` and / or ``query`` parameter. So I made this kind of limitation. Therefore, if the ``place_id`` is used, ``bounds`` and / or ``query`` will be not used. The ``components`` parameter can be used.

Reply:
OK, I see, thank you for the clarification.